.mailmap

```diff
@@ -1,62 +1,176 @@
+
+Jouni K. Seppänen <jks@iki.fi>
+
+Julien Schueller <julien.schueller@gmail.com> <schueller@porsche-l64.phimeca.lan>
+Julien Schueller <julien.schueller@gmail.com> <schueller@bx-l64.phimeca.lan>
+
+Kevin Davies <kdavies4@gmail.com> <daviesk24@yahoo.com>
+
+kikocorreoso <kikocorreoso@gmail.com> <kikocorreoso@users.noreply.github.com>
+
+Klara Gerlei <klarizsofi@gmail.com>
+Klara Gerlei <klarizsofi@gmail.com> klaragerlei <s1466507@sms.ed.ac.uk>
+
+Kristen M. Thyng <kthyng@gmail.com>
+
+Leeonadoh <leo.sunpeng.li@gmail.com>
+
+Lennart Fricke <lennart@die-frickes.eu> <lennart.fricke@kabelmail.de>
+
+Levi Kilcher <levi.kilcher@nrel.gov>
+
+Lion Krischer <krischer@geophysik.uni-muenchen.de>
+
+Martin Fitzpatrick <martin.fitzpatrick@gmail.com> <mfitzp@abl.es>
+
+Matthew Emmett <memmett@gmail.com>
+Matthew Emmett <memmett@gmail.com> <memmett@unc.edu>
+
+Matthias Bussonnier <bussonniermatthias@gmail.com>
+
+matthias <maluethi@protonmail.ch>
+matthias <maluethi@protonmail.ch> <matthias.luethi@lhep.unibe.ch>
+
+Michael Droettboom <mdboom@gmail.com> <mdroe@stsci.edu>
 Michael Droettboom <mdboom@gmail.com> Michael Droettboom <mdboom@debian-vm>
+
+Michiel de Hoon <mjldehoon@yahoo.com>
 Michiel de Hoon <mjldehoon@yahoo.com> Michiel de Hoon <mdehoon@mad002s-MacBook-Air.local>
 Michiel de Hoon <mjldehoon@yahoo.com> Michiel de Hoon <mdehoon@michiel-de-hoons-computer.local>
 Michiel de Hoon <mjldehoon@yahoo.com> Michiel de Hoon <mdehoon@Michiels-MacBook-Pro.local>
 Michiel de Hoon <mjldehoon@yahoo.com> Michiel de Hoon <mdehoon@tkx294.genome.gsc.riken.jp>
+
+MinRK <benjaminrk@gmail.com>
 MinRK <benjaminrk@gmail.com> Min RK <minrk@kerbin.local>
-Nelle Varoquaux <nelle.varoquaux@gmail.com> Varoquaux <nelle.varoquaux@gmail.com>
+
+Moritz Beyreuther <random.seed@web.de>
+
+Nelle Varoquaux <nelle.varoquaux@gmail.com>
+
 Nic Eggert <nic.eggert@gmail.com> Nic Eggert <nic@eggert.pw>
 Nic Eggert <nic.eggert@gmail.com> Nic Eggert <nse23@cornell.edu>
-Nicolas P. Rougier <Nicolas.Rougier@inria.fr> Nicolas Rougier <Nicolas.Rougier@inria.fr>
-Paul Ivanov <pivanov314@gmail.com> Paul Ivanov <pi@berkeley.edu>
-Per Parker <wisalam@live.com> Per <wisalam@live.com>
-Per Parker <wisalam@live.com> solvents <wisalam@live.com>
-Peter Würtz <pwuertz@gmail.com> Peter Würtz <pwuertz@googlemail.com>
-Peter Würtz <pwuertz@gmail.com> pwuertz <pwuertz@gmail.com>
-Peter Würtz <pwuertz@gmail.com> pwuertz <pwuertz@googlemail.com>
-Phil Elson <pelson.pub@gmail.com> pelson <pelson.pub@gmail.com>
-Phil Elson <pelson.pub@gmail.com> pelson <philipelson@hotmail.com>
-Phil Elson <pelson.pub@gmail.com> Phil Elson <philipelson@gmail.com>
-Phil Elson <pelson.pub@gmail.com> Phil Elson <philipelson@hotmail.com>
-Scott Lasley <selasley@me.com> selasley <selasley@me.com>
+
+Nicolas P. Rougier <Nicolas.Rougier@inria.fr>
+
+OceanWolf <juichenieder-tigger@yahoo.co.uk>
+
+Patrick Chen <pat.chen@mail.utoronto.ca>
+
+Paul Hobson <pmhobson@gmail.com>
+Paul Hobson <pmhobson@gmail.com> vagrant <vagrant@precise32.(none)>
```

Comment:
Oof. Sorry about that.

I'm ok with that commit not getting attributed to me.

The main thing I'm concerned about is that y'all (the core team) have as few headaches as possible. I'm on board with whatever you chose to do wit this.